resources/docdb-snapshots.go

```diff
@@ -0,0 +1,96 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/docdb"
+	docdbtypes "github.com/aws/aws-sdk-go-v2/service/docdb/types"
+
+	"github.com/ekristen/libnuke/pkg/registry"
+	"github.com/ekristen/libnuke/pkg/resource"
+	"github.com/ekristen/libnuke/pkg/types"
+
+	"github.com/ekristen/aws-nuke/v3/pkg/nuke"
+)
+
+const DocDBSnapshotResource = "DocDBSnapshot"
+
+func init() {
+	registry.Register(&registry.Registration{
+		Name:     DocDBSnapshotResource,
+		Scope:    nuke.Account,
+		Resource: &DocDBSnapshot{},
+		Lister:   &DocDBSnapshotLister{},
+	})
+}
+
+type DocDBSnapshotLister struct{}
+
+func (l *DocDBSnapshotLister) List(ctx context.Context, o interface{}) ([]resource.Resource, error) {
+	opts := o.(*nuke.ListerOpts)
+	svc := docdb.NewFromConfig(*opts.Config)
+	var resources []resource.Resource
+
+	paginator := docdb.NewDescribeDBClusterSnapshotsPaginator(svc, &docdb.DescribeDBClusterSnapshotsInput{})
+	for paginator.HasMorePages() {
+		page, err := paginator.NextPage(ctx)
+		if err != nil {
+			return nil, err
+		}
+
+		for i := 0; i < len(page.DBClusterSnapshots); i++ {
+			tags, err := svc.ListTagsForResource(ctx, &docdb.ListTagsForResourceInput{
+				ResourceName: page.DBClusterSnapshots[i].DBClusterSnapshotArn,
+			})
+			if err != nil {
+				continue
```

Comment:
Don't forget about this tags continue as well.

Reply:
Ops! my bad